docs/standards/lsp-background/erc725.md

```diff
@@ -14,69 +13,78 @@ sidebar_position: 1
+|        4         | [`DELEGATECALL`](https://eips.ethereum.org/EIPS/eip-7) | Runs the function from another contract, but use the context of the current contract.                               |
+
+## ERC725Y Generic Data Key/Value Store
+
+Using smart contracts to store data on the blockchain can be beneficial when that data will be used or referenced by other smart contracts.
+
+One limitation of this approach is that once a smart contract is deployed with a specific set of variables containing data, it cannot be modified to include new variables. This can be a problem if additional data needs to be stored in the future, as it would require deploying a new version of the contract.
+
+For example, if a token contract is deployed with variables for the token name, symbol, and decimals, it would not be possible to add new variables for the creators or future collaborators.
 
+What is needed is a dynamic way to attach data to a contract even after it has been deployed.
 
+### Specification
 
-### ERC725Y - Generic Data Key/Value Store
+ERC725Y solves this problem by standardizing a mapping of **data keys** to **data values** to store data dynamically, and to have the ability to add or remove data across time without the need of redeploying the contract.
+
+Instead of representing the variable with a specific name such as `age`, or `name`, etc .. They are referenceed with `bytes32`. Same as solidity treat variables under the hood.
 
 :::note
 
 See the section **["Layout of State Variables in Storage"](https://docs.soliditylang.org/en/v0.8.11/internals/layout_in_storage.html)** in the Solidity documentation for more information about the structure of smart contract storage.
 
 :::
 
-This sub-standard enables the account to hold arbitrary data through a generic data key/value store. It gives flexibility to the contract storage. Developers can access the data stored in the contract via data keys instead of referencing the storage slot where the data resides.
+#### Normal contract Vs ERC725Y
 
-![ERC725Y Generic Data Key/Value Store](../../../static/img/standards/erc725/erc725y.jpeg)
+![Smart contract vs ERC725Y](../../../static/img/standards/erc725/SmartContractVsERC725Y.jpeg)
 
-As defined below, the data keys are represented as **bytes32** and the data values are represented as **bytes**. This allows more flexibility and support for complex variable types.
-  
-Difference between Standard Smart Contract Storage and ERC725Y - Generic Data Key/Value Storage:
+Thanks to ERC725Y, contracts become more interoperable, as their storage is represented in the same way. Contracts and interfaces can then read and write data from or to the storage in the same manner via the functions `getData(...)` and `setData(...)`.
 
-![ERC725Y Generic Data Key/Value Store](../../../static/img/standards/erc725/erc725y-difference.jpeg)
```

Comment:
it looks like you don't use this image anymore - can you also delete this image from the repo?